pkg/system/names.go

```diff
@@ -16,8 +16,17 @@ limitations under the License.
 
 package system
 
+import "os"
+
 const (
-	// Namespace holds the K8s namespace where our serving system
-	// components run.
-	Namespace = "knative-serving"
+	NamespaceEnvKey = "SYSTEM_NAMESPACE"
 )
+
+// Namespace holds the K8s namespace where our build system
```

Comment:
```suggestion
// Namespace holds the K8s namespace where our serving system
```

Reply:
Done.
The current CI test need the default namespaces `knative-serving`. So I keep the default namespaces.